src/coreclr/vm/peimage.cpp

```diff
@@ -764,7 +774,7 @@ PTR_PEImageLayout PEImage::CreateLoadedLayout(bool throwOnFailure)
     return pLoadLayout;
 }
 
-PTR_PEImageLayout PEImage::CreateFlatLayout()
+PTR_PEImageLayout PEImage::CreateFlatLayout(BOOL bPermitWriteableSections)
```

Comment:
`bPermitWriteableSections` argument name does not make sense for what the implementation does with it.